.travis.yml

```diff
@@ -59,7 +59,8 @@ jobs:
 
 cache:
   directories:
-    - $HOME/.ivy2
+    - $HOME/.cache/coursier
```

Comment:
Not using Coursier/sbt 1.3 in 1.5.x...  But that could change with #2513 so I thought I'd just keep this part of the backport in.